assets/constants.ts

```diff
@@ -5976,8 +5976,9 @@ export default {
   // 'subtopic URL fragment'.
   "MAX_CHARS_IN_SUBTOPIC_URL_FRAGMENT": 25,
   // This represents the maximum number of characters in the URL fragment for
-  // the blog post.
-  "MAX_CHARS_IN_BLOG_POST_URL_FRAGMENT": 65,
+  // the blog post. It is generated using blog post title which can be 65
+  // characters long and the blog post id which is 12 characters long.
```

Comment:
Is there no separator character between the two parts?

Also, if the ID part is system generated then it's better to have the constant only be for the human-wrItten part.

Reply:
Yeah there is a hyphen in between. We include the ID as well as we use the constant to validate the 'GET' requests made from blog homepage to get blog post page data using URL:
![image](https://user-images.githubusercontent.com/68547101/204086805-5ff39a9e-33a6-455f-9c54-e6bf8f7aca0b.png)